resources/views/components/input/checkbox.blade.php

```diff
@@ -8,7 +8,8 @@
             'class' => "filled-in checkbox-color",
             'name' => $id
         ])}}
-        @checked(old($id) || $checked)
+        @checked(in_array($attributes['value'], (old($id)['option'] ?? []))
```

Comment:
Have you tested this with the other checkbox instances?

Reply:
Yes; with the "remember me" field and voting, it seems to work.